frontend/ui/wikipedia.lua

```diff
@@ -95,6 +95,11 @@ function Wikipedia:getWikiServer(lang)
     return string.format(self.wiki_server, lang or self.default_lang)
 end
 
+-- Say who we are to Wikipedia (see https://meta.wikimedia.org/wiki/User-Agent_policy)
```

Comment:
Incidentally, what did it say before? Some standard LuaSec thing? Nothing?

Reply:
Just `LuaSocket 3.0-rc1`.
Which wasn't causing any issue with Wikipedia, because it's a little known library.

But doing the same kind of multiple images download with "python-requests" was causing many images requests to just get some HTML error message with that URL.

So, we'll be fine if/whenever `LuaSocket` gets a little more noticed by Wikipedia.